unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/RPC/ServerBuilder/RPCServerBuilder.cs

```diff
@@ -26,6 +26,7 @@ public static RpcServer<RPCContext> BuildDefaultServer(RPCContext context, ITran
                    .RegisterService(CRDTServiceImpl.RegisterService)
                    .RegisterService(TransportServiceImpl.RegisterService)
                    .RegisterService(EmotesRendererServiceImpl.RegisterService)
+                   .RegisterService(SceneControllerServiceImpl.RegisterService)
```

Comment:
In a future polishing, this should be only registered for ports starting with `scene-` to scope different RPC to different functions

Reply:
thx for the reminder, tackled at https://github.com/decentraland/unity-renderer/pull/3700/commits/bbb7a318c4cecd2db605021cfdd9e3010d7610a7